README.md

```diff
@@ -19,6 +19,21 @@ This action generates a changelog based on your Git history using [git-cliff](ht
 
 - `OUTPUT`: Output file. (Default: `"git-cliff/CHANGELOG.md"`)
 
+### Important Notes
+- It is important to checkout the repositories as:
```

Comment:
Maybe we can use the GitHub admonition syntax here. ("warning" or "important')

Reply:
I've implemented the requested changes.